src/opera_rtc_s1_browse/create_browse.py

```diff
@@ -17,6 +18,28 @@
 s3 = boto3.client('s3')
 
 
+def now() -> datetime.datetime:
+    return datetime.datetime.now()
```

Comment:
I assume we can rely on `datetime.now` defaulting to UTC if we're running in a Lambda?

Reply:
Whoops, I meant utcnow(), fixed.

The best approach would be `now(tz=datetime.timezone.utc)`, but that creates a little extra work to get `Z` for the timezone in the xml file instead of the `+00:00` output by `isoformat()`. I suspect GIBS can take either, but I don't have any way to check and their examples use `Z`.